NutritionManagement/src/main/Main.java

```diff
@@ -2,19 +2,101 @@
 
 //import io.InputParser;
 //import main.Food;
-//import java.util.ArrayList;
+import java.util.*;
 //import io.OutputFormatter;
 //import main.NutritionValue;
 
+import exceptions.DayNotSelectedException;
+import exceptions.InsufficientArgumentsException;
+import io.InputParser;
+import io.OutputFormatter;
+
 public class Main {
 
-	public static void main(String[] args) {
-		// TODO Auto-generated method stub
-//		  String filePath = "path/to/your/input/file.txt";
-//        ArrayList<Food> foods = InputParser.parseInputFoodFile(filePath);
-//        OutputFormatter.printTotalNutritionValue(totalNutritionValue);
-//        OutputFormatter.printAverageNutritionValue(averageNutritionValue);
+    public static void main(String[] args) {
+        // TODO Auto-generated method stub
+        // String filePath = "path/to/your/input/file.txt";
+        // ArrayList<Food> foods = InputParser.parseInputFoodFile(filePath);
+        // OutputFormatter.printTotalNutritionValue(totalNutritionValue);
+        // OutputFormatter.printAverageNutritionValue(averageNutritionValue);
+
+        Scanner in = new Scanner(System.in);
```

Comment:
Consider closing it in a finally block or using try-with-resources to avoid potential resource leaks.